### ISSUE TITLE
Problem parsing Titles containing brackets () 

### ISSUE THREAD

**Issue by rgoraya** [rgoraya/wc_master#33]
Adding Issue with Wikipedia title containing brackets encounters error when Media wiki call is made to retrieve the content:
For example if we attempt to add an issue named "Defender (association football)", we get the following error in javaScript.
- Uncaught Error: Syntax error, unrecognized expression: (association football)

Duplicate report: _Trying to create the issue uplift (geology), I found the search ended without any results or error message._

**Comment by duyetln** [comment]
I'll see if I can fix it.

**Comment by rgoraya** [comment]
Thanks !

**Comment by duyetln** [comment]
Fixed. Code will be updated soon.
The problem is that in the 'getJson' function, in the 'else' clause, Jval is expected to be a html doc object. But due to the way 'JData' is parsed, it's not always the case. Jval could totally be an innocent string which contains parentheses which will cause $(Jval) to fails. This is where the error comes from.

So to fix this, I modified the code a little bit to parse JData in a cleaner way instead of using a loop.
